WooCommerce/Classes/ViewRelated/Products/ProductsSplitViewCoordinator.swift

```diff
@@ -231,10 +231,12 @@ extension ProductsSplitViewCoordinator: UINavigationControllerDelegate {
     func navigationController(_ navigationController: UINavigationController, didShow viewController: UIViewController, animated: Bool) {
         if didNavigateFromTheLastSecondaryViewControllerToProductListInCollapsedMode(navigationController, didShow: viewController) {
             if let contentType = contentTypes.last, case let .productForm(product) = contentType, let product {
-                ServiceLocator.productImageUploader.startEmittingErrors(
-                    key: .init(siteID: product.siteID,
-                               productOrVariationID: .product(id: product.productID),
-                               isLocalID: false))
+                let uploader = ServiceLocator.productImageUploader
+                let key = ProductImageUploaderKey(siteID: product.siteID,
+                                                  productOrVariationID: .product(id: product.productID),
+                                                  isLocalID: false)
+                uploader.startEmittingErrors(key: key)
+                uploader.sendBackgroundUploadNoticeIfNeeded(key: key, using: ServiceLocator.noticePresenter)
```

Comment:
Similar to the other comment. Moving the product image related code to a helper method will improve readability.

Reply:
Done in f91615d40e15b4ecc5a196c2863e9f29d064940c.